crates/rust-analyzer/src/to_proto.rs

```diff
@@ -688,6 +691,10 @@ pub(crate) fn goto_definition_response(
     }
 }
 
+fn outside_workspace_annotation(snap: &GlobalStateSnapshot) -> Option<String> {
+    snap.config.change_annotation_support().then(|| String::from("OutsideWorkspace"))
```

Comment:
Otoh, this is an std method, so not using it goes against the grain of the language... yeah, I guess it’s better to use it rather than die a death of thousand fights over it :)

Reply:
The poor function already had enough of a fight with its name trying to get into std